framework/auth/oauth_scopes.py

```diff
@@ -71,6 +77,9 @@ class ComposedScopes(object):
     USERS_READ = (CoreScopes.USERS_READ,)
     USERS_WRITE = USERS_READ + (CoreScopes.USERS_WRITE,)
 
+    USERS_FULL_READ = USERS_READ + (CoreScopes.USERS_ADDON_READ,)
```

Comment:
`full_read`and `full_write` scopes can be requested by third parties (outside the OSF).

Unless there is a very strong use case to give third parties power over addons, we should limit addon management endpoints to the `osf.admin` scope. (eg, can only be managed via the OSF)

For example, a one-time-only script could be run by taking the user's cookie value. (which is unrestricted access)

Oauth tokens are generally intended to be issued to third parties, and scopes limit what power those parties have on behalf of that user. I'd recommend restricting this to admin to start; it's much harder to take away power than it is to give it.

@brianjgeiger  , is there such a use case?